package.json

```diff
@@ -12,7 +12,7 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "format": "prettier --write '**/*.{css,html,js,json,jsx,md,mdx,ts,tsx,yaml,yml}'",
+    "format": "prettier --write 'docs/*.{md,mdx}'",
```

Comment:
:memo: https://classic.yarnpkg.com/lang/en/docs/package-json/#toc-scripts

Reply:
> To format a file in-place, use --write. (Note: This overwrites your files!)

https://prettier.io/docs/en/cli.html

> Otherwise, the entry is resolved as a glob pattern using the [glob syntax from the fast-glob module](https://github.com/mrmlnc/fast-glob#pattern-syntax).  

https://prettier.io/docs/en/cli.html

> src/**/*.{css,scss} — matches all files in the src directory (any level of nesting) that have the .css or .scss extension.  

https://github.com/mrmlnc/fast-glob#advanced-syntax